hasjob/__init__.py

```diff
@@ -47,9 +45,8 @@
 
 RQ(app)
 
-baseframe.init_app(app, requires=['hasjob'],
-    ext_requires=['baseframe-bs3',
-        ('jquery.autosize', 'jquery.sparkline', 'jquery.liblink', 'jquery.wnumb', 'jquery.nouislider'),
+baseframe.init_app(app, requires=['baseframe-bs3',
+        'jquery.autosize', 'jquery.sparkline', 'jquery.liblink', 'jquery.wnumb', 'jquery.nouislider',
```

Comment:
We stopped using `sparkline` a while ago. Safe to remove now.

Reply:
Removed